src/main/scala/is/hail/methods/CalculateConcordance.scala

```diff
@@ -63,7 +63,7 @@ object CalculateConcordance {
     left.requireUniqueSamples("concordance")
     right.requireUniqueSamples("concordance")
 
-    val overlap = left.sampleIds.toSet.intersect(right.sampleIds.toSet)
+    val overlap = left.stringSampleIds.toSet.intersect(right.stringSampleIds.toSet)
```

Comment:
Does anything depend on these being keys here?  Can we just use colKeys?

Reply:
I remember something did, but will investigate.

Reply:
it's totally hardcoded for strings. out of scope